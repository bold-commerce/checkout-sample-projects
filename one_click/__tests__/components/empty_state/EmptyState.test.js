import React from 'react';
import EmptyState from '../../../client/components/OneClickLayout/components/EmptyState/EmptyState';
import { render } from '@testing-library/react';

describe('EmptyState', () => {
    test('renders Empty State component', () => {
        const { asFragment } = render( <EmptyState title="Example Title" /> );
        expect(asFragment()).toMatchSnapshot();
    });
});