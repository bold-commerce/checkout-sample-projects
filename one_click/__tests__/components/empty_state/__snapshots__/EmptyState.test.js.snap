// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState renders Empty State component 1`] = `
<DocumentFragment>
  <div
    class="FieldSet--EmptyState"
  >
    <svg
      fill="none"
      height="29"
      viewBox="0 0 28 29"
      width="28"
      xmlns="http://www.w3.org/2000/svg"
    >
      <rect
        fill="#C4C4C4"
        height="5"
        rx="2.5"
        width="28"
      />
      <rect
        fill="#C4C4C4"
        height="5"
        rx="2.5"
        width="28"
        y="8"
      />
      <rect
        fill="#C4C4C4"
        height="5"
        rx="2.5"
        width="28"
        y="16"
      />
      <rect
        fill="#C4C4C4"
        height="5"
        rx="2.5"
        width="28"
        y="24"
      />
      <circle
        cx="2.5"
        cy="10.5"
        fill="white"
        r="1.5"
      />
      <circle
        cx="2.5"
        cy="2.5"
        fill="white"
        r="1.5"
      />
      <circle
        cx="2.5"
        cy="18.5"
        fill="white"
        r="1.5"
      />
      <circle
        cx="2.5"
        cy="26.5"
        fill="white"
        r="1.5"
      />
    </svg>
    Example Title
  </div>
</DocumentFragment>
`;
